command/agent/command.go

```diff
@@ -135,6 +136,12 @@ func (c *Command) readConfig() *Config {
 			return nil
 		}
 
+		// The user asked us to load some config here but we didn't find any,
+		// so we'll complain but continue.
+		if current == nil || reflect.DeepEqual(current, &Config{}) {
```

Comment:
Hm, if they specify a directory and nothing is found, the client/server probably won't load correctly anyways? Maybe we just make it an error